Source/Scene/PostProcessStage.js

```diff
@@ -335,25 +333,32 @@ define([
                 }
                 return undefined;
             }
-        },
-        /**
-         * Whether or not this post process stage requires a depth texture. If <code>true</code> and the WEBGL_depth_texture
-         * extension is unavailable, this stage will not execute.
-         *
-         * @memberof PostProcessStage.prototype
-         * @type {Boolean}
-         * @readonly
-         *
-         * @see {Context#depthTexture}
-         * @see {@link http://www.khronos.org/registry/webgl/extensions/WEBGL_depth_texture/|WEBGL_depth_texture}
-         */
-        requiresDepthTexture : {
-            get : function() {
-                return depthTextureRegex.test(this._fragmentShader);
-            }
         }
     });
 
+    var depthTextureRegex = /uniform\s+sampler2D\s+depthTexture/g;
+
+    /**
+     * Whether or not this post process stage is supported.
+     * <p>
+     * A post process stage is not supported when it requires a depth texture and the WEBGL_depth_texture extension is not
+     * supported.
+     * </p>
+     *
+     * @param {Context} context The context.
+     * @return {Boolean} Whether this post process stage is supported.
+     *
+     * @see {Context#depthTexture}
+     * @see {@link http://www.khronos.org/registry/webgl/extensions/WEBGL_depth_texture/|WEBGL_depth_texture}
+     */
+    PostProcessStage.prototype.isSupported = function(context) {
```

Comment:
Since it's part of the public API it should probably take `scene` instead.

Annoyingly, there will probably need to be a duplicate function that takes `context`, for internal use.